src/ngraph/runtime/cpu/CMakeLists.txt

```diff
@@ -165,6 +165,10 @@ if (NGRAPH_CPU_ENABLE)
         add_dependencies(cpu_backend ext_halide)
     endif()
 
+    if (NGRAPH_DISABLE_FAST_MATH)
+        target_compile_definitions(cpu_backend PRIVATE EIGEN_FAST_MATH=0)
```

Comment:
I tried to set EIGEN_FAST_MATH=1. It doesn't change anything on the result somehow.